_data/jobs.yml

```diff
@@ -1,29 +1,29 @@
-- title: Full Stack Developer (m/f/d)
-  link: https://epages-gmbh.jobs.personio.de/job/384247
-  description: Be part of a cross-functional team and use your Java and/or TypeScript skills to further develop our new ecommerce platform.
-  location: Hamburg
+- title: Software Developer - 3rd Party Integration, REST API (m/f/d)
+  link: https://epages-gmbh.jobs.personio.de/job/202173
+  description: Be part of a Scrum team and support us in the conception, development and maintenance of our connectors and the extension of our REST API.
+  location: Hamburg/Barcelona
 
-- title: Senior Java Developer (m/f/d)
-  link: https://epages-gmbh.jobs.personio.de/job/202115
-  description: Bring your knowledge of Java and Kotlin to work on a microservice-based architecture for our new ecommerce platform.
-  location: Hamburg
+- title: UI Designer HTML/CSS (m/f/d)
+  link: https://epages-gmbh.jobs.personio.de/job/224109
+  description: Drive the visual identity of our products and improve their performance and usability by working closely in an agile team.
+  location: Hamburg/Barcelona
 
 - title: Senior JavaScript Engineer (m/f/d)
   link: https://epages-gmbh.jobs.personio.de/job/201909
   description: Build a responsive, modular, and highly user interactive web application with us that is easy to scale and to extend.
-  location: Hamburg
+  location: Hamburg/Barcelona
 
 - title: Ruby Developer (m/f/d)
   link: https://epages-gmbh.jobs.personio.de/job/202168
   description: With your knowledge of Ruby and JavaScript, you'll support our Rails team to further develop our crucial in-house applications.
-  location: Bilbao
+  location: Hamburg/Barcelona
 
 - title: Technical Product Owner (m/f/d)
   link: https://epages-gmbh.jobs.personio.de/job/226186
   description: Be responsible for the further development of our software and support several Scrum teams with your knowledge.
-  location: Hamburg
+  location: Hamburg/Barcelona
 
-- title: User Experience Designer (m/f/d)
-  link: https://epages-gmbh.jobs.personio.de/job/203174
-  description: Keep an overview of our products' UX and develop user-friendly interaction concepts while working closely with our product management team.
-  location: Hamburg
\ No newline at end of file
+- title: Internship Webdesign (m/f/d) - 6 months or 1 year
```

Comment:
@AliceStatler Just to me sure, here "6 months or 1 year" is not part of the Job title, right? Just an additional information.